ui/packages/components/src/RunResult.tsx

```diff
@@ -1,46 +1,91 @@
 'use client';
 
-import { usePrettyJson } from '@inngest/components/hooks/usePrettyJson';
+import { useMemo } from 'react';
+import z from 'zod';
 
 import { CodeBlock } from './CodeBlock';
-import type { Result } from './types/functionRun';
 
 type Props = {
   className?: string;
-  result: Result;
-  isSuccess?: boolean;
+  result: string;
+  isSuccess: boolean;
 };
 
 export function RunResult({ className, result, isSuccess }: Props) {
-  const prettyResult = result.data && usePrettyJson(result.data);
+  const parsedResult = useParsedResult({ isSuccess, result });
```

Comment:
**question**: Before this, we used `result.data` and now `useParsedResult` is using `result` - is this due to the way that the different API endpoint returns the response? Or is the string `result` include the `data` key? 

I ask this because I'm looking for it to be the same experience, and not have the successful output nested within `{ "data": { "message": "cool" }` - Confirming it should show to the user as `{ "message": "cool" }.

Reply:
We had to make this change because the normal run resolver returns unstructured output data. The old, non-parallelizable one returned structured data. However, there are edge cases the structured resolver didn't handle.

I don't see nested data when I test it:
![image](https://github.com/user-attachments/assets/9ee33ceb-684d-431e-bd35-57e19bec29d9)

Reply:
Ok great! Just wanted to confirm the output wasn't nested as we've had regressions in the past.